internal/exec/vendor_utils.go

```diff
@@ -72,6 +70,17 @@ func ExecuteVendorPullCommand(cmd *cobra.Command, args []string) error {
 	if component != "" && len(tags) > 0 {
 		return fmt.Errorf("either '--component' or '--tags' flag can to be provided, but not both")
 	}
```

Comment:
@coderabbitai aren't these also in conflict with `--everything`? E.g. we shouldn't be able to specify component and/or tags with `--everything`

Reply:
Ah, I see we handle that here: https://github.com/cloudposse/atmos/pull/768/files#diff-35bbd3669178afb0acaaebfb4ec924ec99189338a9bc05232b4f92a53f261518R81